### PR TITLE
addin back button for quiz page for prev question

### PR DESCRIPTION
**Pull request by KennethT404**
Changes in Serve_quiz.py  to add in logic for back button 

added in update_curr_question in db.py. Will make this addition in SQLCRUD

### CODE REVIEW

File: application/app.py [hunk 1]
```diff
@@ -40,25 +40,25 @@
 app.register_blueprint(serve_quiz_blueprint)
 
 # app.register_blueprint(registration_blueprint)
-app.register_blueprint(login_blueprint)
+# app.register_blueprint(login_blueprint)
 
-mail = Mail(app)
+# mail = Mail(app)
```
Comment (kawangwong) [inline application/app.py:45]: Is this intentional/?

File: application/app.py [hunk 2]
```diff
@@ -40,25 +40,25 @@
 app.register_blueprint(serve_quiz_blueprint)
 
 # app.register_blueprint(registration_blueprint)
-app.register_blueprint(login_blueprint)
+# app.register_blueprint(login_blueprint)
```
Comment (kawangwong) [inline application/app.py:43]: Is this intentional/?

**Review by kawangwong — CHANGES_REQUESTED** [review]

File: application/app.py (hunk 2 above)
Comment (KennethT404) [reply application/app.py:43]: yes, view below for the reason

**Comment by KennethT404** [comment]
Commented out logic for the login/session was for me to gain access to the quiz page on the app.

File: application/app.py (hunk 1 above)
Comment (KennethT404) [reply application/app.py:45]: look below in the thread for the reason

**Pull request merged**